src/sites/twitch-twilight/modules/chat/input.jsx

```diff
@@ -647,21 +647,49 @@ export default class Input extends Module {
 
 
 	// eslint-disable-next-line class-methods-use-this
-	sortFavorites(results) {
-		if (!this.chat.context.get('chat.tab-complete.prioritize-favorites')) {
-			return results;
+	sortEmotes(emotes, input) {
+		const preferFavorites = this.chat.context.get('chat.tab-complete.prioritize-favorites');
+		const lowerCaseInput = input.toLowerCase();
+
+		const locale = new Intl.Collator();
```

Comment:
We don't need to construct new collator instances every time we call sortEmotes. Not a huge deal but every little bit of work less we can do in a function that Twitch natively calls *per keypress* is better.

Reply:
Addressed in 0610175.